public/pages/AnomalyCharts/components/AnomaliesStat/AnomalyStat.tsx

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React, { Fragment } from 'react';
+import { EuiIcon, EuiLink, EuiToolTip } from '@elastic/eui';
+import { Monitor } from '../../../../models/interfaces';
+import { getAlertingMonitorListLink } from '../../../../utils/utils';
+import { formatNumber } from '../../../utils/helpers';
+
+interface AnomalyStatProps {
+  title: any;
+  description: any;
+}
+
+export const AnomalyStat = (props: AnomalyStatProps) => {
+  return (
+    <div className="euiStat euiStat--leftAligned">
+      <div className="euiText euiText--small euiStat__description">
+        {props.description}
+      </div>
+      <div
+        className="euiTitle euiTitle--medium euiStat__title"
+        style={{ display: 'inline' }}
+      >
+        {props.title}
+      </div>
+    </div>
+  );
```

Comment:
Any specific reason why we are unable to use Stats component directly ?

Reply:
EUI stat can only support text , if you check the UI capture, we need to add some EUI icon to the description.

Reply:
You can login to our test domain and check AD result page, we need to add some link in Stat title and description.